core/func.go

```diff
@@ -35,45 +35,43 @@ type MultiFn struct {
 	Methods []Fn
```

Comment:
Nitpick:  are these really methods?  Unless I'm missing something, I think these are just "Functions".

Reply:
oh, I named this `Methods` with [Julia lingo](https://docs.julialang.org/en/v1/manual/methods/) in mind (or more generally, multiple dispatch). Julia has concept of `function` which refers to the name itself (For example `+`).. But when you actually invoke a function `+` with arguments, a `Method` is chosen based on the type & number of arguments passed.